### ISSUE TITLE
Can you provide a version in Objective C ?

### ISSUE THREAD

**Issue by mmarshak** [ParsePlatform/AnyPhone#7]
Thanks

**Comment by gfosco** [comment]
Would consider merging a PR if someone else wanted to add an Objective-C version, but not too interested in adding it myself.

**Comment by Ge0rges** [comment]
Yeah a Objective-C version would be awesome!

**Comment by madhavc** [comment]
@gfosco Would you consider an Android App PR? and would it be under this repo?

**Comment by gfosco** [comment]
Absolutely.  Put the android app in an android folder and send a PR against
this repo.  We welcome any and all contributions and enhancements..
Supporting more phone number formats, country selector, more platforms,
etc. :) Thanks!

**Comment by Geronimobile** [comment]
Any update on this? It would be extremely helpful. Thanks!

**Comment by LtrDan** [comment]
+1 for Objective-C

**Comment by nlutsenko** [comment]
Would gladly review an ObjC implementation of this :grin:

**Comment by madhavc** [comment]
Thinking of writing ObjC implementation!
